packages/fakta-fordel-beregningsgrunnlag/src/components/tilkommetAktivitet/TilkommetAktivitetUtils.ts

```diff
@@ -1,12 +1,150 @@
-/* eslint-disable import/prefer-default-export */
 import { Inntektsforhold, VurderInntektsforholdPeriode } from '@navikt/ft-types/src/beregningsgrunnlagFordelingTsType';
+import { ForlengelsePeriodeProp } from '@navikt/ft-types/src/beregningsgrunnlagTsType';
+import { ArbeidsgiverOpplysningerPerId, Beregningsgrunnlag } from '@navikt/ft-types';
+import { AktivitetStatus } from '@navikt/ft-kodeverk';
+import erPeriodeTilVurdering from '../util/ForlengelseUtils';
 
-export const getInntektsforhold = (
-  inntektsforholdPerioder?: VurderInntektsforholdPeriode[],
-): Inntektsforhold | undefined =>
-  inntektsforholdPerioder
-    ?.flatMap(inntektsforholdPeriode => inntektsforholdPeriode.inntektsforholdListe)
-    .filter(
-      (inntektsforhold, i, liste) =>
-        liste.findIndex(listeElement => listeElement.arbeidsgiverId === inntektsforhold.arbeidsgiverId) === i,
-    )[0];
+export const getInntektsforhold = (inntektsforholdPerioder?: VurderInntektsforholdPeriode[]): Inntektsforhold[] =>
+  inntektsforholdPerioder === undefined
+    ? []
+    : inntektsforholdPerioder
+        ?.flatMap(inntektsforholdPeriode => inntektsforholdPeriode.inntektsforholdListe)
+        .filter(
+          (inntektsforhold, i, liste) =>
+            liste.findIndex(listeElement => listeElement.arbeidsgiverId === inntektsforhold.arbeidsgiverId) === i,
+        );
+
+function unike() {
```

Comment:
Hvorfor function og ikke const? Har ikke noe godt argument for const, men liker at alt er likt :P

Reply:
Det er defaulten til intelliJ for autogenerering av funksjoner, og så har eg tenkt at forskjellen ikkje er stor nok til at eg har endret på det.